Documentation/contributor-guide/triage_issues.md

```diff
@@ -48,9 +48,42 @@ Adding a `type` label to an issue helps create visibility on the health of the p
 
 ### Support requests
 
-Some people mistakenly use GitHub issues to file support requests. Usually they are asking for help operating or configuring some aspect of etcd. In general we prefer that support requests are raised as [discussions](https://github.com/etcd-io/etcd/discussions), however if you identify that an issue is a support request please add the `type/support` or `type/question` label.
+As a general rule the focus for etcd support is to address common themes in a broad way that helps all users, i.e. through channels like known issues, frequently asked questions and high quality documentation. To make the best use of project members time we should avoid providing 1:1 support if a broad approach is available.
 
-When responding to these requests please keep in mind the etcd [supported versions policy](https://etcd.io/docs/v3.6/op-guide/versioning/#service-versioning-and-supported-versions). If the request relates to an unsupported version and will require significant effort please instead kindly make the issue creator aware of the etcd support policy, encourage they update to a supported release, then close the issue, refer to [this example](https://github.com/etcd-io/etcd/issues/15796).
+Some people mistakenly use our GitHub bug report or feature request templates to file support requests. Usually they are asking for help operating or configuring some aspect of etcd. Support requests for etcd should instead be raised as [discussions](https://github.com/etcd-io/etcd/discussions).
+
+Common types of support requests are:
+
+ 1. Questions about configuring or operating existing well documented etcd features, for example <https://github.com/etcd-io/etcd/issues/15836>. Note - If an existing feature is not well documented please apply the `area/documentation` label and propose documentation improvements that would prevent future users from stumbling on the problem again.
```

Comment:
Definition of "well documented feature" would be useful :P. 

For me the shown example is not well documented as responses community provided where based on examples and not by a link to documentation. Using examples usually means that it is based on personal knowledge and not official source. 

My criteria for well documented is whether we are able to answer question with section of documentation present in https://etcd.io/docs/. In thi scase searching etcd.io for `--endpoints` I get nothing. 

For me this issue requires a followup to create a command line flag documentation like K8s does https://kubernetes.io/docs/reference/kubectl/#operations or https://github.com/kubernetes-sigs/metrics-server/blob/master/docs/command-line-flags.txt.

This should be pretty simple to autogenerate.

Reply:
I think a definition for a well documented feature is a whole separate markdown file 😂.  I don't want this doc to get too verbose so I suggest we leave it at that for now. 

I agree that my example wasn't the best, I've updated to another issue which is more clearly fully covered by existing documentation. Let me know if you have any suggestions for a better example.